modules/nw-sriov-network-attachment.adoc

```diff
@@ -126,14 +126,14 @@ spec:
   networkNamespace: <target_namespace> <4>
   vlan: <vlan> <5>
   spoofChk: "<spoof_check>" <6>
+  linkState: <link_state> <7>
+  maxTxRate: <max_tx_rate> <8>
+  minTxRate: <min_rx_rate> <9>
+  vlanQoS: <vlan_qos> <10>
+  trust: "<trust_vf>" <11>
+  capabilities: <capabilities> <12>
```

Comment:
To my knowledge, we do not test these with VMs. Continuing the discussion on the BZ https://bugzilla.redhat.com/show_bug.cgi?id=1895007